sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosTableSchemaInferer.scala

```diff
@@ -5,122 +5,125 @@ package com.azure.cosmos.spark
 import com.azure.cosmos.CosmosAsyncClient
 import com.azure.cosmos.models.CosmosQueryRequestOptions
 import com.fasterxml.jackson.databind.JsonNode
-import com.fasterxml.jackson.databind.node.{ArrayNode, BinaryNode, BooleanNode, DecimalNode, DoubleNode,
-    FloatNode, IntNode, LongNode, NullNode, ObjectNode, TextNode}
-
 
 // scalastyle:off underscore.import
+import com.fasterxml.jackson.databind.node._
+
 import org.apache.spark.sql.types._
+
 import scala.collection.JavaConverters._
 // scalastyle:on underscore.import
 
 // Infers a schema by reading sample data from a source container.
 private object CosmosTableSchemaInferer
-    extends CosmosLoggingTrait {
-    val RAW_JSON_BODY_ATTRIBUTE_NAME = "_rawBody"
-    private val TIMESTAMP_ATTRIBUTE_NAME = "_ts"
-    private val ID_ATTRIBUTE_NAME = "id"
-    private val defaultSchemaForInferenceDisabled = StructType(Seq(
-        StructField(RAW_JSON_BODY_ATTRIBUTE_NAME, StringType),
-        StructField(ID_ATTRIBUTE_NAME, StringType),
-        StructField(TIMESTAMP_ATTRIBUTE_NAME, LongType)
-    ))
-
-    def inferSchema(inferredItems : Seq[ObjectNode]): StructType = {
-        if (inferredItems.isEmpty){
-            // No documents to infer from
-            StructType(Seq())
-        }
+  extends CosmosLoggingTrait {
 
-        // Create a unique map of all distinct properties from documents
-        val uniqueStructFields = inferredItems.foldLeft(Map.empty[String, StructField])({
-            case (map, item) => inferDataTypeFromObjectNode(item) match {
-                case Some(mappedList) =>
-                    map ++ mappedList.map(mappedItem => {
-                        if (map.contains(mappedItem._1) && map(mappedItem._1).dataType != mappedItem._2.dataType){
-                            // If 2 documents contain the same property name but different type, we default to String
-                            (mappedItem._1, StructField(mappedItem._1, StringType))
-                        }
-                        else{
-                            mappedItem
-                        }
-                    })
-            }
-        })
-
-        StructType(uniqueStructFields.valuesIterator.toSeq)
-    }
+  private[spark] val RAW_JSON_BODY_ATTRIBUTE_NAME = "_rawBody"
```

Comment:
scala naming convention for constants is Camel Case
please see this:
https://docs.scala-lang.org/style/naming-conventions.html

Reply:
ACK - will changes this

Reply:
Fixed in next iteration